### DISCUSSION TITLE
Can't get --exclude-id-file to work

### DISCUSSION THREAD

**Discussion by InPlotITrust** [Serene-Arc/bulk-downloader-for-reddit#820] [Q&A]
Hello,

If I use `--exclude-id` and provide a single id it ignores that id, but I can't get the `--exclude-id-file` option to work. 

I've extracted the id's of successful downloads using the provided script.

This is the download command I'm running

>bdfr download "J:\Test" --subreddit MySubreddit --file-scheme "{DATE}\_{POSTID}\_{REDDITOR}\_{FLAIR}\_{TITLE}" -S new --exclude-id-file "J:\Test\succeed.txt" --log "J:\Test\log.txt"

Not sure if I should see anything in the log about loading the file, but there's nothing in the log about loading the id's present in the file.

Am I overlooking something?

Thanks for your time.

**Comment by Serene-Arc** [comment]
What is the format of the IDs in the file?

**Reply by InPlotITrust** [reply]
Hey,

Sorry for late response. IRL stuff and forgot to check back.

The format of the ID's  is the reddit id
![image](https://user-images.githubusercontent.com/65138051/233457756-dd67352e-a97a-4072-8523-ee0b5626b129.png)
